src/rb3keyboard.c

```diff
@@ -121,12 +145,12 @@ static inline void handle_key(MIDIDRUM *MIDI_DRUM, unsigned char key) {
             MIDI_DRUM->notedown(MIDI_DRUM->sequencer, MIDI_DRUM->channel,
                                                     48 + key + 12 * MIDI_DRUM->octave,
                                                     MIDI_DRUM->velocity);
+        } else {
+            MIDI_DRUM->noteup(MIDI_DRUM->sequencer, MIDI_DRUM->channel,
+                                                    48 + key + 12 * MIDI_DRUM->octave,
+                                                    0);
         }
     }
-    if (!(MIDI_DRUM->key_state & (1 << (24 - key)))) {
-        MIDI_DRUM->noteup(MIDI_DRUM->sequencer, MIDI_DRUM->channel,
-                                            48 + key + 12 * MIDI_DRUM->octave, 0);
-    }
```

Comment:
Adding back these lines seems to fix the issue, why was this removed, was there a specific reason?

Reply:
I think I changed it as I was getting Note Off for every key (unpressed) constantly in aseqdump.